test/client/fixtures/runner/cross-domain/in-iframe-test.js

```diff
@@ -1,11 +1,20 @@
-var hammerhead = window.getTestCafeModule('hammerhead');
-var hhsettings = hammerhead.get('./settings').get();
+var hammerhead   = window.getTestCafeModule('hammerhead');
+var hhsettings   = hammerhead.get('./settings').get();
+var browserUtils = hammerhead.utils.browser;
 
 var testCafeRunner = window.getTestCafeModule('testCafeRunner');
 var RunnerBase     = testCafeRunner.get('./runner-base');
 
 hhsettings.serviceMsgUrl = '/ping/10';
 
+// HACK: we shouldn't override native dialogs methods, while testing on the android simulator, because
+// this results in an unexpected window.alert call (https://github.com/DevExpress/testcafe/issues/471).
```

Comment:
this issue is complex, so it would be better to add a link to the related comment (https://github.com/DevExpress/testcafe/issues/471#issuecomment-220386269)